### PR TITLE
[SPARK-34621][SQL] Unify output of ShowCreateTableAsSerdeCommand  and ShowCreateTableCommand

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
Unify output of ShowCreateTableAsSerdeCommand  and ShowCreateTableCommand

### Why are the changes needed?
Unify output of ShowCreateTableAsSerdeCommand  and ShowCreateTableCommand

### Does this PR introduce _any_ user-facing change?
No

### CODE REVIEW

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala [hunk 1]
```diff
@@ -297,13 +297,13 @@ case class DropNamespace(
...
     extended: Boolean,
-    override val output: Seq[Attribute] = DescribeNamespace.getOutputAttr) extends Command {
+    override val output: Seq[Attribute] = DescribeNamespace.getOutputAttrs) extends Command {
   override def children: Seq[LogicalPlan] = Seq(namespace)
 
 }
 
 object DescribeNamespace {
-  def getOutputAttr: Seq[Attribute] = Seq(
+  def getOutputAttrs: Seq[Attribute] = Seq(
```
Comment (AngersZhuuuu) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala:306]: When check the code found name not consistence. Just change in this pr or make a followup? @cloud-fan

Comment (cloud-fan) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala:306]: It's very trivial. I'm OK to fix the typo here.

**Review by cloud-fan — APPROVED** [review]

**Comment by AngersZhuuuu** [comment]
retest this please

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Pull request closed without merging**